osu.Game.Rulesets.Taiko/UI/TaikoRulesetContainer.cs

```diff
@@ -77,6 +104,25 @@ private void loadBarLines()
             }
         }
 
+        protected override bool OnKeyDown(InputState state, KeyDownEventArgs args)

```

Comment:
See previous comment: implement `IKeyBindingHandler<TaikoAction>` on Playfield and use that instead.